services/workflows-service/src/workflow/workflow.controller.internal.ts

```diff
@@ -197,6 +199,21 @@ export class WorkflowControllerInternal {
     }
   }
 
+  @common.Put(':id/properties/:documentId')
```

Comment:
An endpoint to update a document by id should suffice. We also update a document's type and category for example.